adapters/cwire/cwiretest/exemplary/banner.json

```diff
@@ -0,0 +1,125 @@
+{
+  "mockBidRequest": {
+    "id": "80ce30c53c16e6ede735f123ef6e32361bfc7b22",
+    "cur": [
+      "CHF"
+    ],
+    "imp": [
+      {
+        "id": "102",
+        "banner": {
+          "h": 250,
+          "w": 300,
+          "pos": 0
+        }
+      }
```

Comment:
Can we add an extra test case or JSON test file that comes with more `req.imp[i].ext` fields?

Reply:
Added